CHANGELOG.md

```diff
@@ -14,31 +16,53 @@ might need to be adjusted and either explicitly name the positional
 argument or resolve the type ambiguity by casting to the intended type.
 
 **Features**
+* PG15 support added
+* MultiNode High Avalibility improvements
```

Comment:
```suggestion
```

These should be honored with a proper textual explanation in the free text above